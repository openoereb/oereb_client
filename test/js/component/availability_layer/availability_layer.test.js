import VectorLayer from 'ol/layer/Vector';
import React from "react";
import {render} from '@testing-library/react'
import {Provider} from "react-redux";

import OerebAvailabilityLayer from
  "../../../../oereb_client/static/src/component/availability_layer/availability_layer";
import MainStore from "../../../../oereb_client/static/src/store/main";

describe('availability layer component', () => {

  let component;

  beforeEach(() => {
    const layer = new VectorLayer();
    component = render(
      <Provider store={MainStore}>
        <OerebAvailabilityLayer availabilityLayer={layer} />
      </Provider>
    );
  });

  it('should render availability layer element', () => {
    expect(component.asFragment()).toMatchSnapshot();
  });

});