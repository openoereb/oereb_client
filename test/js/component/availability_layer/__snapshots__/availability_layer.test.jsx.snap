// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`availability layer component > should render availability layer element 1`] = `<DocumentFragment />`;
